File: client/src/app/shared/wifi-details.ts
export class WifiDetails {
  constructor(in_use: boolean,
              bssid: string,
              ssid: string,
              mode: string,
              channel: string,
              rate: string,
              signal: string,
              security: string) {
    this.in_use = in_use;
    this.bssid = bssid;
    this.ssid = ssid;
    this.mode = mode;
    this.channel = channel;
    this.rate = rate;
    this.signal = signal;
    this.security = security;
  }

  in_use: boolean;
  bssid: string;
  ssid: string;
  mode: string;
  channel: string;
  rate: string;
  signal: string;
  security: string;
}

File: wifi-setup-service/wifi_details.py
class WifiDetails:
    def __init__(self, in_use: bool, bssid: str, ssid: str, mode: str, channel: int, rate: str, signal: int, security: str):
        self.in_use = in_use
        self.bssid = bssid
        self.ssid = ssid
        self.mode = mode
        self. channel = channel
        self. rate = rate
        self.signal = signal
        self.security = security

    in_use: bool
    bssid: str
    ssid: str
    mode: str
    channel: int
    rate: str
    signal: int
    security: str
